src/test/java/edu/kis/powp/jobs2d/events/StarCommandFactory.java

```diff
@@ -0,0 +1,20 @@
+package edu.kis.powp.jobs2d.events;
+
+import java.util.ArrayList;
+import java.util.List;
+import edu.kis.powp.jobs2d.command.*;
+
+public class StarCommandFactory {
+    public static ImmutableComplexCommand createStarCommand() {
```

Comment:
Return type ImmutableComplexCommand should be substituted with a more general one. ICompoundCommand seems optimal